src/Services/Kros.Organizations.Api/appsettings.Development.json

```diff
@@ -9,5 +9,10 @@
 
   "UserRole": {
     "AuthServiceUrl": "http://localhost:9002/api/Permissions"
+  },
+
+  "ApiJwtAuthorization": {
```

Comment:
Tieto pridane nastavovacky treba dat aj do appsettings.json aby sa mohli nahradzat v release pipeline...

Reply:
Už to tam bolo, ale vymazal som hodnoty a nechal len kľúče.